server/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import { CallForHelpServer } from './server';
+
+const app = new CallForHelpServer();
+
+export { app }
```

Comment:
Unnecessary export

Reply:
The reason I'm exporting the app is for future purposes. if we need to incorporate the use of express in the app then the server will initialize the express HTTP server and we export the server object to use it whenever we need to.
also, it can be used to then use things the server object holds that are not only the express, if any will arise in the future.

Reply:
There's this YAGNI principle, but you do you